rust/src/domains/polars/series/mod.rs

```diff
@@ -189,19 +203,22 @@ impl SeriesDomain {
     /// If the domain of elements is of type `AtomDomain<T>`, then returns the domain as that type,
     /// otherwise returns an error.
     pub fn atom_domain<T: 'static + CheckAtom>(&self) -> Fallible<&AtomDomain<T>> {
+        self.element_domain::<AtomDomain<T>>()
+    }
+
+    /// # Proof Definition
+    /// If the domain of elements is of type `D`, then returns the domain as that type,
+    /// otherwise returns an error.
+    pub fn element_domain<D: 'static>(&self) -> Fallible<&D> {
```

Comment:
Does this need to be public? If possible, it's nice to keep the API small, so the user has an easier time finding the method they actually need.

Reply:
It can be public-- I was going to expose this same method in FFI.